### ISSUE TITLE
Unregister-PSFConfig doesn't function when using FileUserLocal

### ISSUE THREAD

**Issue by affieuk** [PowershellFrameworkCollective/psframework#167]
This below code works if you remove the `-Scope` parameter and it's value, defaulting to the registry

```
PS C:\> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.1.0-preview.4
PSEdition                      Core
GitCommitId                    6.1.0-preview.4
OS                             Microsoft Windows 10.0.17134
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0


PS C:\> Get-PSFConfig -Module test
PS C:\> $configFile = "$Env:LocalAppData\PowerShell\PSFramework\Config\psf_config.json"
PS C:\> Test-Path $configFile
False
PS C:\> Set-PSFConfig -FullName test.1234.name -Value joe -Validation string
PS C:\>
PS C:\> Get-PSFConfig -Module test | Register-PSFConfig -Scope FileUserLocal
PS C:\> Test-Path $configFile
True
PS C:\> Get-PSFConfig -Module test | Unregister-PSFConfig
PS C:\> Test-Path $configFile
True
PS C:\> Unregister-PSFConfig -Module test
PS C:\> Test-Path $configFile
True
PS C:\> Unregister-PSFConfig -Module test -Scope FileUserLocal
PS C:\> Test-Path $configFile
True
PS C:\> Get-Content $configFile
{
  "Value": "joe",
  "Type": 6,
  "Style": "Default",
  "FullName": "test.1234.name",
  "Version": 1
}
```

**Comment by FriedrichWeinmann** [comment]
Dang! Thanks for pointing this out, I can reproduce this and am going try and fix that soonest

**Comment by FriedrichWeinmann** [comment]
Heya @affieuk ,

thank you again for reporting this! The fix has been published in release 0.9.24.91 that has just hit the Gallery!

Cheers,
Fred
